### PR TITLE
✨ updated to use links in repo, not in my fork

### PR DESCRIPTION
**Pull request by clubanderson**
## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by clubanderson** [comment]
@MikeSpreitzer updated links to point to main for now, and updated release.md to require update to release number. please review

File: scripts/create-kubestellar-demo-env.sh [hunk 1]
```diff
@@ -58,10 +58,11 @@ echo -e "KubeStellar Version: ${kubestellar_version}"
 
 echo -e "Checking that pre-req softwares are installed..."
 if [ "$k8s_platform" == "kind" ]; then
-    curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
-else
-    curl -s https://raw.githubusercontent.com/clubanderson/kubestellar/refs/heads/add-k3d-to-create-demo-env/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker k3d
     # curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
+    curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/main/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
```
Comment (MikeSpreitzer) [inline scripts/create-kubestellar-demo-env.sh:62]: This one does not need to be changed, right? The released check_pre_req.sh is as good as the one in `main` for all but "k3d", right?

Comment (clubanderson) [reply scripts/create-kubestellar-demo-env.sh]: updated

**Review by MikeSpreitzer — APPROVED** [review]
/lgtm
/approve

**Pull request merged**